src/Utility/FieldFunctions.C

```diff
@@ -415,6 +463,46 @@ RealArray FieldsFuncBase::getValues (const Vec3& X)
 }
 
 
+std::vector<Real> FieldsFuncBase::getGradient (const Vec3& X) const
+{
+  const Vec4* x4 = dynamic_cast<const Vec4*>(&X);
+  if (!x4)
+    return {};
+  else
+  {
```

Comment:
Also to be consistent with the other methods, remove the else {} here. Logic will be unchanged anyway.

Reply:
yeah, stupid copy-paste thingie. removed.